publish_courses/migrations/0001_initial.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+LTI_PERMISSIONS_DATA = [
+    ('publish_courses', '*', 'AccountAdmin', True),
```

Comment:
I checked canvas.dev and I'm showing up as an `Account Admin` -- maybe another person who's not Canadian can double-check that and we can remove this line (7).